src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.spec.ts

```diff
@@ -100,6 +100,10 @@ describe('OsdListComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('should have columns that are searchable', () => {
```

Comment:
"searchable" or "sortable" (as the PR description suggests)?

Reply:
done